CHANGELOG.md

```diff
@@ -11,6 +11,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1740** - `cumulus_meta.workflow_start_time` is now set in Cumulus
   messages
 
+### Added
+
+- **Ability to set custom backend API url in the archive module**
+  - Add `api_url` definition in `tf-modules/cumulus/archive.tf`
+  - Add `archive_api_url` variable in `tf-modules/cumulus/variables.tf`
+- **Fix default values for urs_url in variables.tf files**
```

Comment:
Sorry, this one should still be under the "Fixed" header. but the other ones should stay here.

Reply:
my bad - updated in a95c093